ci/release_notes.md

```diff
@@ -0,0 +1,6 @@
+# Fixes
+- acceptance tests are now run in parallel
+
+# Acknowledgements
+
+Thanks @peterellisjones for the PR
```

Comment:
Do we need a release for this? We don't change anything in the functionality, therefore I wouldn't create one.

Reply:
Good point